src/main/java/org/jabref/logic/formatter/bibtexfields/CleanupUrlFormatter.java

```diff
@@ -38,16 +39,16 @@ public String format(String value) {
 
         Matcher matcher = PATTERN_URL.matcher(value);
         if (matcher.find()) {
-        toDecode = matcher.group(1);
-
+            toDecode = matcher.group(1);
         }
         try {
             decodedLink = URLDecoder.decode(toDecode, StandardCharsets.UTF_8.name());
-        }
-        catch (UnsupportedEncodingException e) {
+        } catch (UnsupportedEncodingException e) {
             LOGGER.warn("Used unsupported character encoding", e);
         }
-        return decodedLink;
+
+        String result = new RemoveLatexCommandsFormatter().format(decodedLink);
```

Comment:
Although somewhat of an edge case, but doesn't this lead to problems with the valid url `jabref.org/test%5Ctest` (which I guess would be reduced to `jabref.org/test` instead of the expected `jabref.org/test\test`?

Reply:
Not this LOC in particularly, but in general: Yes. I added a test case for that.

In the fetcher itself, I use the RemoveLatexCommandsFormatter directly. I did not know how to convert a LayoutFormatter to a Formatter. Thus, I "manually" updated the field in cbb3836. Moments later, I worked on `LayoutFormatterBasedFormatter` which wraps a `LayoutFormatter` in a `Formatter` and IMHO made the code more nice.

Reply:
I'm still unsure if we should really remove latex commands from the URL. I guess this is only an issue for the DBLP fetcher and nowhere else.

Reply:
Oh, now I got it. Reverted that in https://github.com/JabRef/jabref/pull/5582/commits/83cbb3d02ce679ce6f6e5daf88d8510b3a804ac3